pkg/provisioners/inmemory/dispatcher/main.go

```diff
@@ -14,9 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// A sidecar that implements filtering of Cloud Events sent out via HTTP. Implemented as an HTTP
-// proxy that the main containers need to write through.
-
 package main
```

Comment:
Leave this file in `cmd/`

Reply:
Yes. I'll move both controller and dispatcher to cmd